frontend/device/kobo/device.lua

```diff
@@ -675,6 +676,12 @@ function Kobo:init()
             -- Libra 2 w/ a BD71828 PMIC
             self.power_dev = "/dev/input/by-path/platform-bd71828-pwrkey-event"
         end
+                
+        if util.fileExists("/dev/input/by-path/platform-bd71828-pwrkey.4.auto-event") then
```

Comment:
While it makes little difference in practice, I presume that should be written like this to be more analogous to all the others:
```suggestion
        elseif util.fileExists("/dev/input/by-path/platform-bd71828-pwrkey.4.auto-event") then
```